src/drivers/render/render-driver.js

```diff
@@ -14,7 +17,16 @@ export const makeRenderDriver = (videoRender) => {
 
 	const unsetCanvas = () => videoRenderInstance = null;
 
-	return sink_ =>{
+	const click_ = fromEvent('intersect', videoRender.events);
```

Comment:
This is just intersect detection and not a click.
Idea:
Try to refactor into a function like this:
const events = (type) => fromEvent(type, videoRender.events)